Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -30,9 +30,9 @@
     <Reference Include="System.Web" />
   </ItemGroup>
   
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.2'">
     <PackageReference Include="Microsoft.PowerShell.Commands.Utility">
-      <Version>6.1.0</Version>
+      <Version>6.1.0-rc.1</Version>

```

Comment:
The CVE showed up again even after installing PowerShell.Utility 6.1.0. This time it's updated to the specific version recommended by the fix: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2018-8292